src/services/item/plugins/tag/controller.test.ts

```diff
@@ -203,6 +207,67 @@ describe('Tag Endpoints', () => {
           payload: { name: 'name', category: TagCategory.Discipline },
         });
 
+        expect(response.statusCode).toBe(StatusCodes.FORBIDDEN);
+      });
+    });
+  });
+  describe('DELETE /:itemId/tags/:tagId', () => {
+    it('Throw for invalid item id', async () => {
+      const response = await app.inject({
+        method: HttpMethod.Delete,
+        url: `/items/invalid/tags/${v4()}`,
+      });
+      expect(response.statusCode).toBe(StatusCodes.BAD_REQUEST);
+    });
+    it('Throw for invalid tag id', async () => {
+      const response = await app.inject({
+        method: HttpMethod.Delete,
+        url: `/items/${v4()}/tags/invalid}`,
+      });
+      expect(response.statusCode).toBe(StatusCodes.BAD_REQUEST);
+    });
```

Comment:
Should these be in the signed in section ?

Reply:
It does not even check authentication, so it's valid for both unauthenticated and authenticated users.

Reply:
As discussed just add more context before the test.